src/api/deposit/DepositApi.ts

```diff
@@ -1,7 +1,7 @@
 import BN from 'bn.js'
-import { assert } from '@gnosis.pm/dex-js'
+import { assert, toBN } from '@gnosis.pm/dex-js'
```

Comment:
Importing `toBN` from `dex-js` is ok here because it ultimately comes from `web3` which is a peer dep for `dex-js` and therefore local `web3` is used. But I would be wary of just importing any utility function because of potential version conflictsor duplicating deps.

For example if we ever need a function that makes use of `bignumber.js` and import it by the way of `dex-js`, it would be good to move `bignumber.js` to peer deps too.